samcli/local/lambdafn/runtime.py

```diff
@@ -170,6 +169,12 @@ def invoke(
         timer = None
         container = None
         try:
+            # Track runtime
```

Comment:
Do you think a decorator still makes sense here?

Reply:
I don't have strong preference towards decorator or just inline tracking. 
My main concern was mainly that the `capture_parameter` decorator was not very intuitive to read and use. 
I think we can think more about how the decorators for tracking metric should be like. Meanwhile, it looks more intuitive to me to use `track_metric` inline here